src/docsBrowser.ts

```diff
@@ -33,16 +34,40 @@ export namespace DocsBrowser {
         panel = window.createWebviewPanel('haskell.showDocumentationPanel', ttl, ViewColumn.Beside, {
           localResourceRoots: [Uri.parse(documentationDirectory)],
           enableFindWidget: true,
```

Comment:
I thought we had this enabled but I just tried this out and can't seem to get it to work either. Is this a vs-code bug?